Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/compositeeditor/CompositeEditorModel.java

```diff
@@ -1110,12 +1109,6 @@ public DataTypeInstance validateComponentDataType(int rowIndex, String dtString)
 		DataTypeManager originalDTM = getOriginalDataTypeManager();
 		DataType newDt = DataTypeHelper.parseDataType(rowIndex, dtString, this, originalDTM,
 			provider.dtmService);
-		if (newDt == null) {
-			if (dt != null) {
-				throw new UsrException("No data type was specified.");
-			}
-			throw new AssertException("Can't set data type to null.");
-		}
```

Comment:
This part can never be reached, as `newDt == null` is always `false`. If an exception is thrown during `parseDataType`, this will not be reached. `parseDataType` will not return null, in this case an exception would have been thrown before in https://github.com/NationalSecurityAgency/ghidra/blob/master/Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/compositeeditor/DataTypeHelper.java#L102